resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+                :placeholder="$t('settings.application.logo.hint')"
+                v-model="settings.logo"
+                :disabled="isBusy || viewOnly"
+                class="my-2"
+                :state='fieldState("logo")'
+              >
+              </b-form-input>
+            </b-input-group>
+            <b-form-text v-if="!viewOnly">{{ $t('settings.application.logo.uploadTitle') }}</b-form-text>
+            <b-input-group v-if="!viewOnly">
+              <b-form-file
+                id="application-logo-form-file"
+                :state='fieldState("logo_file")'
+                :disabled="isBusy"
+                :browse-text="$t('app.browse')"
+                :placeholder="$t('settings.application.logo.selectFile')"
+                v-model="uploadLogoFile"
+                accept="image/jpeg, image/png, image/gif, image/svg+xml"
+              >
+              </b-form-file>
+              <template #append v-if="uploadLogoFile">
+                <b-button id="application-upload-button" variant="danger" @click="uploadLogoFile = null">
+                  <b-icon-x></b-icon-x>
+                </b-button>
+              </template>
+            </b-input-group>
+          </b-col>
+        </b-row>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('logo')"></div>
+          <div v-html="fieldError('logo_file')"></div>
+        </template>
+      </b-form-group>
+
+      <!--Room limit settings-->
+      <b-form-group
+        label-for="application-room-limit-input"
+        :description="$t('settings.application.roomLimit.description')"
+        :state='fieldState("room_limit")'
+      >
+
+        <template v-slot:label>
+          {{ $t('settings.application.roomLimit.title') }}
+        </template>
+
+        <b-form-radio-group
+          class='mb-2'
+          id="application-room-limit-radio-group"
+          v-model='roomLimitMode'
+          :options='roomLimitModeOptions'
+          :disabled='isBusy || viewOnly'
+          :state='fieldState("room_limit")'
+          @change="roomLimitModeChanged"
+          stacked
+        ></b-form-radio-group>
+
+        <b-form-input
+          id='application-room-limit-input'
+          type='number'
+          :state='fieldState("room_limit")'
+          v-model='settings.roomLimit'
+          min='0'
+          :disabled='isBusy || viewOnly'
+          v-if="roomLimitMode === 'custom'">
+        </b-form-input>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('room_limit')"></div>
+        </template>
+      </b-form-group>
+
+      <!--Pagination page size settings-->
+      <b-form-group
+        label-for="application-pagination-page-size-input"
+        :description="$t('settings.application.paginationPageSize.description')"
+        :state='fieldState("pagination_page_size")'
+      >
+
+        <template v-slot:label>
+          {{ $t('settings.application.paginationPageSize.title') }}
```

Comment:
Why we need the extra template? The label could be set in the form group through an attribute!